scripts/setup/ci_setup_osx.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# Travis CI uses this script to set up the test environment on macOS.
+
+# Install and start Mosquitto for MQTT pull request builds.
+if [ "$TRAVIS_PULL_REQUEST" != "false" ] && [ "$RUN_TEST" = "mqtt" ]; then
+    # Only update brew if we need it, and make it quiet.
+    brew update > /dev/null
```

Comment:
Could you check if everything needs to be redirected?  It may be only stdout and stderr should still be printed.

Reply:
I believe `>` only redirects stdout and stderr still gets printed. Removing the line though so that brew will not update every time.